bot/seasons/valentines/lovecalculator.py

```diff
@@ -0,0 +1,119 @@
+          "this relationship to work, and put time and effort into it, meaning spending time together, talking to "
+          "each other etc., than nothing shall stand in your way.",
+    '20': "The chance of this relationship working is not very high. You both need to put time and effort into this "
+          "relationship, if you want it to work out well for both of you. Talk with each other about everything and "
+          "don't lock yourself up. Spend time together. This will improve the chances of this relationship's survival "
+          "by a lot",
+    '5': "There might be a chance of this relationship working out somewhat well, but it is not very high. With a lot "
+         "of time and effort you'll get it to work eventually, however don't count on it. It might fall apart quicker "
+         "than you'd expect.",
+    '1': "The chance of this relationship working out is really low. You can get it to work, but with high costs and "
+         "no guarantee of working out. Do not sit back, spend as much time together as possible, talk a lot with each "
+         "other to increase the chances of this relationship's survival "
+}
+
+
+class LoveCalculator:
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(aliases=('love_calculator', 'love_calc'))
+    async def love(self, ctx, name_one: discord.Member, name_two: discord.Member):
+        love_meter = (name_one.id + name_two.id) % 100
+        if love_meter >= 95:
+            love_status = LOVE_STAT['95'][randint(0, len(LOVE_STAT['95']))]
+            love_stat_i = '95'
+        elif 80 <= love_meter < 90:
+            love_status = LOVE_STAT['80']
+            love_stat_i = '80'
+        elif 60 <= love_meter < 80:
+            love_status = LOVE_STAT['60']
+            love_stat_i = '60'
+        elif 45 <= love_meter < 60:
+            love_status = LOVE_STAT['45']
+            love_stat_i = '45'
+        elif 30 <= love_meter < 45:
+            love_status = LOVE_STAT['30']
+            love_stat_i = '30'
+        elif 20 <= love_meter < 30:
+            love_status = LOVE_STAT['20']
+            love_stat_i = '20'
+        elif 5 <= love_meter < 20:
+            love_status = LOVE_STAT['5']
+            love_stat_i = '5'
+        elif 1 <= love_meter < 5:
+            love_status = LOVE_STAT['1']
+            love_stat_i = '1'
+        else:
+            love_meter = 100
+            love_status = LOVE_STAT['0'][randint(0, len(LOVE_STAT['0']))]
+            love_stat_i = '0'
+
+        embed = discord.Embed(
+            title=love_status,
+            description=f'{name_one.display_name} \u2764 {name_two.display_name} scored {love_meter}%!\n\u200b',
+            color=discord.Color.dark_magenta()
+        )
+        embed.add_field(
+            name='A letter from Dr. Love:',
+            value=LOVE_TXT[love_stat_i]
+        )
+
+        await ctx.message.channel.send(embed=embed)
+
+    @love.error
+    async def love_error(self, ctx, error):
```

Comment:
This command-specific error handler eats all uncaught errors here. It's better if it was removed, as this block is only logging two exceptions and ignoring the rest, when by default we handle missing args already with user feedback and logging in the default error handler elsewhere.

Reply:
Removed.